databases.qmd

```diff
@@ -260,7 +239,7 @@ It's a rather non-traditional introduction to SQL but we hope it will get you qu
 Luckily, if you understand dplyr you're in a great place to quickly pick up SQL because so many of the concepts are the same.
 
 We'll explore the relationship between dplyr and SQL using a couple of old friends from the nycflights13 package: `flights` and `planes`.
-These datasets are easy to get into our learning database because dbplyr has a function designed for this exact scenario:
+These datasets are easy to get into our learning database because dbplyr comes with a function copies the tables from nycflights13 to our database:
```

Comment:
```suggestion
These datasets are easy to get into our learning database because dbplyr comes with a function that copies the tables from nycflights13 to our database:
```